show-changes

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ $# -lt 1 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 OLD [NEW]"
+  echo "Shows the changes between the git references by assembling the changelog/ folder entries"
+  echo "Should be run in the folder that contains the coreos-overlay, portage-stable, and scripts repository folders."
+  echo "The NEW reference can be omitted and will then default to HEAD."
+  exit 1
+fi
+
+OLD="$1"
+NEW="${2-HEAD}"
+
+
+echo "Changes since ${OLD}"
+
+
+for section in security bugfixes changes updates; do
+  echo
+  case "${section}" in
+    security)
+      echo "Security fixes:"
+      ;;
+    bugfixes)
+      echo "Bug fixes:"
+      ;;
+    changes)
+      echo "Changes:"
+      ;;
+    updates)
+      echo "Updates:"
+      ;;
+    *)
+      echo "wrong cases" > /dev/stderr
```

Comment:
I usually use `&2`, so:
```suggestion
      echo "wrong cases" >&2
```

Reply:
I think it's like the long options vs short options discussion :D

Reply:
Touché. :)